.github/workflows/static-checks.yaml

```diff
@@ -66,10 +64,6 @@ jobs:
           sudo -E ./ci/install_yq.sh
         env:
           INSTALL_IN_GOPATH: false
-      - name: Install rust
```

Comment:
Although rust is preinstalled in the free runner, I think we should stick with version in https://github.com/kata-containers/kata-containers/blob/main/versions.yaml#L374

Reply:
Agreed, will do this.

Reply:
Done